calculatorApp/src/app/home/home.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnInit {
+
+  currentNumber = '0';
+  firstOperand = null;
+  operator = null;
+  waitForSecondNumber = false;
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  // pulls first and second number from calculator then sets currentNumber "display"
+  getNumber(v: string) {
+    console.log(v);
+    if (this.waitForSecondNumber) {
+      this.currentNumber = v;
+      this.waitForSecondNumber = false;
+    } else {
+      this.currentNumber === '0' ? this.currentNumber = v : this.currentNumber += v;
+    }
+  }
+
+  // adds decimal to current number
+  getDecimal() {
+    if (!this.currentNumber.includes('.')) {
+      this.currentNumber += '.';
+    }
+  }
```

Comment:
When testing out the calculator, I saw that dividing whole numbers always returned whole numbers. I did however see this logic of returning decimals for decimals. Next step would be to allow decimal returns for dividing whole numbers.